operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakRealmImportStatus.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.operator.v2alpha1.crds;
+
+import java.util.List;
+import java.util.Objects;
+
+import static org.keycloak.operator.v2alpha1.crds.KeycloakRealmImportStatusCondition.DONE;
+
+public class KeycloakRealmImportStatus {
+    private List<KeycloakRealmImportStatusCondition> conditions;
+
+    public List<KeycloakRealmImportStatusCondition> getConditions() {
+        return conditions;
+    }
+
+    public void setConditions(List<KeycloakRealmImportStatusCondition> conditions) {
+        this.conditions = conditions;
+    }
+
+    public boolean isDone() {
+        return conditions
+                .stream()
+                .filter(c -> c.getStatus())
+                .anyMatch(c -> c.getType().equals(DONE));
```

Comment:
In order to not filter first, and then check any Match, shouldnt it be faster doing `.anyMatch(c -> c.getStatus() && c.getType().equals(DONE))` ? wdyt ?

Reply:
no strong opinion here, ok